index.js

```diff
@@ -168,4 +173,30 @@ module.exports = {
     'wrap-iife': ERROR,
     yoda: [WARN, 'never'],
   },
+  overrides: [
+    {
+      files: ['*.ts'],
+      rules: {
+        '@typescript-eslint/array-type': 'warn',
+        '@typescript-eslint/no-var-requires': 'off',
+        '@typescript-eslint/no-use-before-define': 'off',
```

Comment:
Would it really help? If we have a few such files we could ignore such lines, but in other cases, it helps to prevent `undefined` surprises

Reply:
Don't think so